website/src/css/_config.scss

```diff
@@ -0,0 +1,348 @@
+// All of the following configs are used in the website and documentation provided by the Design team Glyph
+
+// Colors
+
+@mixin detox-colors($prefix) {
+  --#{$prefix}-r0: #f14c52ff;
+  --#{$prefix}-y0: #f7ba29ff;
+  --#{$prefix}-b0: #3e77e4ff;
+  --#{$prefix}-u0: #7633e5ff;
+  --#{$prefix}-e0: #3ebf72ff;
+
+  --#{$prefix}-r1a: #f14c5289;
```

Comment:
Actually if we are talking about some serious auto-generation business, you could try to define transparency levels like (1, 0.8. 0.6. 0.5) and then generate all those `r1a, r2a, r3a` in a loop.

Reply:
Hi, thanks again for the detailed explanation. I worked on it and my solution seems to work except for admonitions colors, navbar links colors. Could be more depending on Percy run. Sent in latest commit.